pkgs/development/python-modules/stack-data/default.nix

```diff
@@ -1,55 +1,43 @@
-{ asttokens
-, buildPythonPackage
-, cython
-, executing
+{ buildPythonPackage
 , fetchFromGitHub
-, git
 , lib
-, littleutils
-, pure-eval
-, pygments
+, python3
+, pythonPackages
 , pytestCheckHook
-, setuptools-scm
-, toml
-, typeguard
 }:
 
 buildPythonPackage rec {
   pname = "stack-data";
-  version = "0.2.0";
-  format = "setuptools";
+  version = "0.6.3";
+  pyproject = true;
 
   src = fetchFromGitHub {
     owner = "alexmojaki";
     repo = "stack_data";
     rev = "v${version}";
-    hash = "sha256-brXFrk1UU5hxCVeRvGK7wzRA0Hoj9fgqoxTIwInPrEc=";
+    hash = "sha256-dmBhfCg60KX3gWp3k1CGRxW14z3BLlair0PjLW9HFYo=";
   };
 
-  nativeBuildInputs = [
```

Comment:
The unwritten convention is to keep the inputs in logical order of the process. First build requirement, then runtime requirements.

Reply:
Ok thanks, i think i've wrote inputs in this order. Could you check ?